exchange/exchange.go

```diff
@@ -146,10 +146,14 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 		//If includebrandcategory is present in ext then CE feature is on.
 		if requestExt.Prebid.Targeting != nil && requestExt.Prebid.Targeting.IncludeBrandCategory != nil {
 			var err error
-			bidCategory, adapterBids, err = applyCategoryMapping(ctx, requestExt, adapterBids, *categoriesFetcher, targData)
+			var rejections []string
+			bidCategory, adapterBids, rejections, err = applyCategoryMapping(ctx, requestExt, adapterBids, *categoriesFetcher, targData)
 			if err != nil {
 				return nil, fmt.Errorf("Error in category mapping : %s", err.Error())
 			}
+			for _, message := range rejections {
+				errs = append(errs, fmt.Errorf(message))
```

Comment:
Minor: You're calling `fmt.Errorf` which allows for formatting parameters, but that's never used here. Consider switching to `errors.New`.

Reply:
Updated to use `errors.New` instead